Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN npm run build:docker
 # Install prod dependencies inside the dist directory
 RUN cp package.json dist && cd dist && npm install --only=prod
 
-FROM builder as production
+FROM node:14-slim
 
 COPY . .
```

Comment:
I was thinking about this yesterday, won't we need an alias for the images?

Shouldn't this `COPY` command include a `--from=build`, assuming the alias for the first image is `build`?

Without that won't this command just be copying files from the host?